specs/mediawiki/v1/content.yaml

```diff
@@ -18,28 +18,23 @@ paths:
     get:
       tags:
         - Page content
-      description: List pages
-      x-backend-request:
-        uri: /{domain}/sys/page_revisions/page/
+      description: List page properties
 
-  /{module:page}/{title}:
+  /{module:page}/title/:
     get:
       tags:
         - Page content
-      description: Redirect to the latest HTML for this page
-      # TODO
-      # x-backend-request:
-      #   uri: /{domain}/sys/page_revisions/page/
+      description: List all known pages
 
-  /{module:page}/{title}/:
+  /{module:page}/title/{title}/:
```

Comment:
It sounds like there are multiple implied use cases relating to tracing the past (or future) history of a page, and citing a page at a given point in time.  I recommend we enumerate the user stories to best inform the design of this endpoint.